src/providers/fault-summary/__tests__/fault-summary.spec.ts

```diff
@@ -164,6 +165,72 @@ describe('faultSummaryProvider', () => {
         expect(result[0].faultCount).toEqual(4);
       });
     });
+    describe('Category C', () => {
+      it('should return an empty array if there are no driving faults', () => {
+        const result = faultSummaryProvider.getDrivingFaultsList({}, TestCategory.C);
+        expect(result.length).toEqual(0);
+      });
+      it('should return an array matching the number of driving faults > 0', () => {
+        const data: TestData = {
+          drivingFaults: {
+            useOfMirrorsChangeDirection: 1,
+            useOfMirrorsSignalling: 2,
+          },
+        };
+        const result = faultSummaryProvider.getDrivingFaultsList(data, TestCategory.C);
+        expect(result.length).toEqual(2);
+      });
+      it('should correctly return any manoeuvre faults', () => {
+        const data: CatCUniqueTypes.TestData = {
+          manoeuvres: {
+            reverseLeft: {
+              selected: true,
+              controlFault: 'DF',
+              observationFault: 'DF',
+            },
+          },
+        };
+        const result = faultSummaryProvider.getDrivingFaultsList(data, TestCategory.C);
+        expect(result.length).toEqual(2);
+        expect(result[0].competencyDisplayName).toEqual('Reverse - Control');
+        expect(result[1].competencyDisplayName).toEqual('Reverse - Observation');
+      });
+      it('should correctly return any vehicle checks faults when there are 4 driving faults', () => {
+        const data: CatCUniqueTypes.TestData = {
+          vehicleChecks: {
+            tellMeQuestions: [
+              { outcome: 'DF' },
+              { outcome: 'DF' },
+            ],
+            showMeQuestions: [
+              { outcome: 'DF' },
+              { outcome: 'DF' },
+            ],
+          },
+        };
+        const result = faultSummaryProvider.getDrivingFaultsList(data, TestCategory.C);
+        expect(result.length).toEqual(1);
+        expect(result[0].faultCount).toEqual(4);
+      });
+      it('should correctly return any vehicle checks faults when there are 5 driving faults', () => {
```

Comment:
I'd like this test text to explicitly state that it would expect 4 faults

Reply:
is that not this expectation `expect(result[0].faultCount).toEqual(4);` ?

Reply:
It's more that if someone wasn't familiar with the rule  they might change this to 5. it's just bringing the description into line with the test. So for example "Should correctly return 4 vehicle check faults when there are 5" ....

Reply:
ah right the description, yes valid point will update that now